### DISCUSSION TITLE
Problem displaying graphs when zoomed in

### DISCUSSION THREAD

**Discussion by dstarkovn** [akvorado/akvorado#928] [Q&A]
hi. i have Problem displaying graphs when zoomed in
![image](https://github.com/akvorado/akvorado/assets/17964606/e6ae7704-a1fa-4fca-bddc-04835fe7ef05)

when zoomed:
![image](https://github.com/akvorado/akvorado/assets/17964606/0ef652e0-8335-4829-bd7a-3e2818392cbf)

SNMP values
![image](https://github.com/akvorado/akvorado/assets/17964606/faad62d0-5118-4ac9-b0d3-8c7e65ddd1d9)

**Comment by vincentbernat** [comment]
Currently, Akvorado uses the received time and your device seems to buffer flows every minutes. You can lower the timeout of flows to avoid this issue.

**Reply by dstarkovn** [reply]
Thank you Vincent, I'll try

**Comment by dstarkovn** [comment]
and yet I don’t understand, my interface is a maximum of gigabit, but when you get closer, peaks of 1.5 GB

![image](https://github.com/akvorado/akvorado/assets/17964606/8405552b-9aee-4881-8a3f-b8a209345348)

![image](https://github.com/akvorado/akvorado/assets/17964606/6b9b2573-b19d-4998-b6d2-b533e63c09e4)

![image](https://github.com/akvorado/akvorado/assets/17964606/eae2c83a-2ee3-481d-8738-13c24eae6e15)

**Comment by christo2310** [comment]
@dstarkovn  i am seeing the same behaviour. did you get it working?

**Reply by vincentbernat** [reply]
You may want to use the timestamp from the flow instead of the timestamp of the packet. https://demo.akvorado.net/docs/configuration#flow
